### DISCUSSION TITLE
Parsing by gatherers (sbd_config)

### DISCUSSION THREAD

**Discussion by scmschmidt** [trento-project/wanda#131] [Q&A]
As far as I could see in my tests the gatherers do some processing of the values (e.g. corosync.conf cuts everything after the first space after a non-space character and does trimming.).  
This eases the development of checks significantly!

The sbd_config gatherer seems to do very little in this regard. Since this is a shell (bash) file, the syntax is very strict and nothing much need to be done, but I would make in exception with SBD_DEVICES. This is basically a semicolon-separated list of devices. It is planned to change the return value from string to list of (trimmed) strings?

**Comment by scmschmidt** [comment]
Discussion seems to be started already here: https://github.com/trento-project/wanda/discussions/117#discussion-4633032

**Reply by scmschmidt** [reply]
Must have accidentally marked it as answered myself.

**Reply by fabriziosestito** [reply]
@scmschmidt Please refer to the #117 answer, we prefer using the split string instead of adding an exception only for that field.
